app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -41,9 +41,9 @@ public void onClick(View view) {
                     Toast.makeText(view.getContext(), "What did you click?? :P", Toast.LENGTH_SHORT).show();
                 }
 
-                if (url.equals(null)) {
+                if (url.equals(null) || url.isEmpty()) {
```

Comment:
Didn't notice it. The crash was due to the `url` being empty string so this fixes it.
But this is obviously wrong way to compare `null`.